documentation/modules/con-iot-concepts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-guide.adoc
+
+[id='con-iot-connectivity-{context}']
+= IoT connectivity concepts
+
+IoT connectivity feature enables {ProductName} to be used for managing and connecting devices with backend applications. In a typical IoT application devices are treated differently than ordinary messaging applications. Instead of using arbitrary addresses and security
+configurations, developers want to be able to handle device identities and security configurations explicitly, support multiple protocols often used in the IoT space and provide
+uniform support for expected device communication patterns.
+
+IoT connectivity layer introduces a few more services to make that happen.
+
+image::https://raw.githubusercontent.com/EnMasseProject/enmasse/master/documentation/design/overview/enmasse_iot_view.png[IoT Connectivity]
+
+*Device registry* allows developers to register devices and provide their credentials. With these credentials devices can connect to *protocol adapters*
+using one of the supported protocols (HTTP, MQTT, LoRaWAN and SigFox). Once connected, devices can send and receive messages from backend applications using one of the following messaging semantics:
+
+* *Telemetry* - allows devices to send non-durable data to the backend applications. This means that messages will be sent using the multicast address. This channel is most suited for sending non-critical sensor readings.
+* *Events* - allows devices to send durable data to the backend applications. This means that messages will be sent using the queue address. This channel is most suited for sending more important device data, like alerts, notifications and such.
+
+Backend applications can also send *Command* messages back to devices. Commands can be used to trigger actions on devices. Examples include updating a configuration property, installing a software component or switching the state of an actuator.
```

Comment:
```suggestion
Back-end applications can also send *Command* messages to devices. Commands can be used to trigger actions on devices. Examples include updating a configuration property, installing a software component, or switching the state of an actuator.
```

Reply:
@dejanb, I think "back" is unnecessary in this context/to relate that intended meaning.